bento_authorization_service/config.py

```diff
@@ -27,6 +27,11 @@ class Config(BaseSettings):
     #    the instance-local IdP is supported.
     openid_well_known_url: str = "https://bentov2auth.local/realms/bentov2/.well-known/openid-configuration"
 
+    permitted_token_algorithms: frozenset = frozenset(
+        ["RS256", "RSA-OAEP", "AES"]
```

Comment:
RSA-OAEP is an encryption algorithm not a signing algorithm as far as I can tell. worth checking out https://auth.sd4h.ca/keycloak/realms/bentov2/.well-known/openid-configuration (specifically `id_token_signing_alg_values_supported` which should map pretty well to the access token too) to see which signing algorithms come with keycloak by default and then disabling any youve read are insecure.